.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           su `id -un 1000` -c "./gradlew build && ./gradlew publishToMavenLocal"
 
       - name: Upload Coverage Report
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v1
```

Comment:
Why the downgrade from v3 to v1?

Reply:
Adding @peterzhuamazon for more insights, its because the image used `public.ecr.aws/opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3` does not support `GLIBC_2.27`. https://github.com/nodejs/node/issues/46412